data/styles/clear/include/priorities_4_overlays.prio.txt

```diff
@@ -789,6 +789,7 @@ tourism-resort                                      # icon z16- (also has captio
 === 2750
 
 tourism-motel                                       # icon z16- (also has caption(optional) z16-)
+tourism-zoo-petting                                 # icon z15- and caption z13-14 (also has caption(optional) z15-)
```

Comment:
please remove iconless captions from z13-14

Reply:
also its quite an attraction for local people, isn't it?
I think put it next to `amenity-pharmacy` or `amenity-fountain`

Reply:
fixed :)